services/ui-src/src/verbiage/pages/profile.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  intro: {
+    header: "My Account",
+    body: "If any information is incorrect, please contact the Managed Care Reporting (MCR) Help Desk at",
+    email: {
```

Comment:
Should this text instead say `... ,please contact the Home & Community Based Services (HCBS) Help Desk`? Or do all apps use the MCR Help Desk?

Reply:
I had asked Megan about this and it's the one help desk for all the apps!

Reply:
Good to know 😄